katas/content/single_qubit_gates/index.md

```diff
@@ -175,18 +175,18 @@ $$
 
 This representation can be used to carry out calculations in Dirac notation without ever switching back to matrix representation:
 
-$$X|0\rangle = \big(|0\rangle\langle1| + |1\rangle\langle0|\big)|0\rangle = |0\rangle\langle1|0\rangle + |1\rangle\langle0|0\rangle = |0\rangle\big(\langle1|0\rangle\big) + |1\rangle\big(\langle0|0\rangle\big) = |0\rangle(0) + |1\rangle(1) = |1\rangle$$
+$$X\ket{0} = \big(\ket{0}\bra{1} + \ket{1}\bra{0}\big)\ket{0} = \ket{0}\braket{1|0} + \ket{1}\braket{0|0} = \ket{0}\big(\braket{1|0}\big) + \ket{1}\big(\braket{0|0}\big) = \ket{0}(0) + \ket{1}(1) = \ket{1}$$
```

Comment:
I will note that on the katas on QCOM this looks as below currently

<img width="705" alt="image" src="https://github.com/microsoft/qsharp/assets/993909/152d5902-9067-4c80-b952-567622db028c">

When I build/view this locally I see:

<img width="960" alt="image" src="https://github.com/microsoft/qsharp/assets/993909/8100f047-e3bb-455d-9d42-62efbb702ec0">


I'm not sure why some of the brakets seems to be getting doubled up.

Reply:
Interestingly, I only see that on our MathJax rendering test page, and not the default Katex one, and when I paste the same equation into the MathJax demo page at <https://www.mathjax.org/#demo> it looks fine. So it could just be our test rendering page has an issue. I'll check it out, but seems like a false alarm.

Reply:
Yes, looks like some bug in our treatment of \braket macro, it looks like it takes the whole argument and repeats it in the bra and ket sections